### PR TITLE
Update to uProtocol 1.5.8

### PR DESCRIPTION
**Pull request by dddj698**
Update to uProtocol 1.5.8,
Use the new version 0.1.8,
Remove cloudevent.

### CODE REVIEW

**Comment by dddj698** [comment]
uP-L2 Communication Layer APIs will be done in a separate PR.

File: src/main/kotlin/org/eclipse/uprotocol/transport/validator/UAttributesValidator.kt [hunk 1]
```diff
@@ -162,14 +148,33 @@ sealed class UAttributesValidator {
         }
```
Comment (stevenhartley) [inline src/main/kotlin/org/eclipse/uprotocol/transport/validator/UAttributesValidator.kt:148]: Actually the lowest priority is CS1 and not CS0 per the spec here: https://github.com/eclipse-uprotocol/up-spec/blob/main/basics/qos.adoc#default-priority

**Review by stevenhartley — CHANGES_REQUESTED** [review]
Can you also include the code coverage CI workflow as well for Kotlin? you can copy what Neelam added for up-java in https://github.com/eclipse-uprotocol/up-java/blob/main/.github/workflows/coverage-comment-pr.yml

File: pom.xml [hunk 2]
```diff
@@ -15,35 +15,26 @@
...
         <junit.version>4.12</junit.version>
         <maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
-
         <slf4j.version>2.6.18</slf4j.version>
-
-        <cloudevents.version>2.4.2</cloudevents.version>
-
         <protobuf.version>3.21.10</protobuf.version>
-        <git.tag.name>uprotocol-core-api-1.5.7</git.tag.name>
+        <git.tag.name>main</git.tag.name>
```
Comment (stevenhartley) [inline pom.xml:37]: @dddj698 please update to use v1.6.0-alpha.2 on this PR or the next one but I'll still approve this Pr if I don't find anything else.

File: src/main/kotlin/org/eclipse/uprotocol/transport/validator/UAttributesValidator.kt [hunk 3]
```diff
@@ -1,52 +1,45 @@
...
+import org.eclipse.uprotocol.uri.validator.isRpcResponse
 import org.eclipse.uprotocol.uuid.factory.getTime
 import org.eclipse.uprotocol.uuid.factory.isUuid
 import org.eclipse.uprotocol.v1.*
 import org.eclipse.uprotocol.validation.ValidationResult
 
 
 /**
- * [UAttributes] is the class that defines the Payload. It is the place for configuring time to live, priority,
+ * [UAttributes] is the class that defines the Payload. It is the place
```
Comment (stevenhartley) [inline src/main/kotlin/org/eclipse/uprotocol/transport/validator/UAttributesValidator.kt:26]:
```suggestion
 * [UAttributes] is the class that defines the uProtocol header that includes routing and payload metadata. It is the place
```

File: src/main/kotlin/org/eclipse/uprotocol/transport/validator/UAttributesValidator.kt [hunk 4]
```diff
@@ -1,52 +1,45 @@
...
+ * for configuring time to live, priority,
  * security tokens and more.
- * Each UAttributes class defines a different type of message payload. The payload can represent a simple published
+ * Each UAttributes class defines a different type of message payload. The
+ * payload can represent a simple published
```
Comment (stevenhartley) [inline src/main/kotlin/org/eclipse/uprotocol/transport/validator/UAttributesValidator.kt:30]: @dddj698 can you fix the formatting of the header (the line breaks) in the various methods in this class?

**Review by stevenhartley — CHANGES_REQUESTED** [review]
Minor change then we can merge in, thanks @dddj698

File: pom.xml (hunk 2 above)
Comment (dddj698) [reply pom.xml:37]: it's in next PR

File: src/main/kotlin/org/eclipse/uprotocol/transport/validator/UAttributesValidator.kt (hunk 3 above)
Comment (dddj698) [reply src/main/kotlin/org/eclipse/uprotocol/transport/validator/UAttributesValidator.kt:26]: will do in next PR

File: src/main/kotlin/org/eclipse/uprotocol/transport/validator/UAttributesValidator.kt (hunk 4 above)
Comment (dddj698) [reply src/main/kotlin/org/eclipse/uprotocol/transport/validator/UAttributesValidator.kt:30]: will do in next PR

**Review by stevenhartley — APPROVED** [review]
LGTM

**Pull request merged**